runtime/centrifuge/src/migrations.rs

```diff
@@ -10,9 +10,29 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
+use cfg_primitives::{Balance, PoolId, CFG};
+use cfg_types::tokens::{
+	usdc::{
+		CURRENCY_ID_AXELAR, CURRENCY_ID_DOT_NATIVE, CURRENCY_ID_LOCAL, CURRENCY_ID_LP_ARB,
+		CURRENCY_ID_LP_BASE, CURRENCY_ID_LP_CELO, CURRENCY_ID_LP_ETH, LOCAL_ASSET_ID,
+	},
+	CurrencyId, LocalAssetId,
+};
+
 frame_support::parameter_types! {
 	pub const ClaimsPalletName: &'static str = "Claims";
 	pub const MigrationPalletName: &'static str = "Migration";
+	pub const UsdcVariants: [CurrencyId; 6] = [CURRENCY_ID_DOT_NATIVE, CURRENCY_ID_AXELAR, CURRENCY_ID_LP_ETH, CURRENCY_ID_LP_BASE, CURRENCY_ID_LP_ARB, CURRENCY_ID_LP_CELO];
+	pub const LocalAssetIdUsdc: LocalAssetId = LOCAL_ASSET_ID;
+	pub const LocalCurrencyIdUsdc: CurrencyId = CURRENCY_ID_LOCAL;
+	pub const PoolIdAnemoy: PoolId = 4_139_607_887;
+	pub const PoolCurrencyAnemoy: CurrencyId = CURRENCY_ID_DOT_NATIVE;
+	pub const UsdcDot: CurrencyId = CURRENCY_ID_DOT_NATIVE;
+	pub const UsdcEth: CurrencyId = CURRENCY_ID_LP_ETH;
+	pub const UsdcBase: CurrencyId = CURRENCY_ID_LP_BASE;
+	pub const UsdcArb: CurrencyId = CURRENCY_ID_LP_ARB;
+	pub const UsdcCelo: CurrencyId = CURRENCY_ID_LP_CELO;
+	pub const MinOrderAmount: Balance = 10 * CFG;
```

Comment:
`MinOrderAmount` is measured in `amount_out` used in each `TraidingPair`. In this case, we're adding a lot of decimals from `CFG` and `USDC`, and we're only using 6 decimals, so we can never place a swap.

Reply:
@mustermeiszer do we really need a `TradingPair` min amount if we already have a fulfilled min amount? Can not the fulfilled minimum amount also be used as the min order amount? Do we need to be very precise here?

Reply:
Great catch @lemunozm! I intended to go with `10 * USDC`, e.g. `10_000_000`. Was also wondering about the need of the min order amount.

Reply:
IIRC, it is there to avoid creating orders with 0.0000001 amount and then opening an attack vector, wasting chain resources very cheaply. But having already a min fulfillment amount, I think this issue is automatically solved for both placing an order and filling an order.

But looking forward to what @mustermeiszer says about this

Reply:
You know I'm always in favor of removing things 😆. Now, seriously, I think removing this can simplify many things: usage, migrations, maintenance...

Reply:
Can we do this in a subsequent PR though please?

Reply:
If that is fine, I changed the amount to 10 USDC here: [0be2e66](https://github.com/centrifuge/centrifuge-chain/pull/1713/commits/0be2e66118a52153e5a34e9fae5d969d0adbc236)

Reply:
Make sense!

I can own that in the new PR if we want to go with it.